tests/GetLicenseDataTest.php

```diff
@@ -0,0 +1,42 @@
+<?php
+/**
+ * How to set up and run tests:
+ *
+ * 1. Install PHPUnit via Composer:
+ *    composer require --dev phpunit/phpunit
+ *
+ * 2. Copy the `credentials.example.php` file to `credentials.php` and fill in your actual credentials:
+ *    cp tests/credentials.example.php tests/credentials.php
+ *    // Then edit `tests/credentials.php` to include your actual API credentials.
+ *
+ * 3. Run the tests using PHPUnit:
+ *    vendor/bin/phpunit
+ */
+use PHPUnit\Framework\TestCase;
+
+class GetLicenseDataTest extends TestCase
+{
+    public function testLicenseData()
+    {
+        require_once 'credentials.php';
```

Comment:
Please see my suggestion above, we can do it from a `bootstrap.php` file under `tests` directory.